src/coremidi.cs

```diff
@@ -177,13 +190,25 @@ interface MidiCIProfileState : NSSecureCoding
 		[Export ("disabledProfiles")]
 		MidiCIProfile[] DisabledProfiles { get; }
 
+		[Deprecated (PlatformName.iOS, 14, 0, message : "Use 'ctr (byte midiChannel, MidiCIProfile[] enabled, MidiCIProfile[] disabled)' instead.")]
```

Comment:
`.ctor` is how the metadata is named

Reply:
I think something like this would be even better:

> "Use the '(byte midiChannel, MidiCIProfile[] enabled, MidiCIProfile[] disabled)' constructor instead."

Reply:
We use the `.ctor (Foo, Bar)` on other places but not opposed to change or to use

> "Use the '.ctor (byte midiChannel, MidiCIProfile[] enabled, MidiCIProfile[] disabled)' overload instead".

Reply:
I used ctr because I did see it in other places, I have no preference.. I'll take @rolfbjarne option